transport-native-io_uring/src/test/java/io/netty/channel/uring/PollRemoveTest.java

```diff
@@ -81,7 +81,7 @@ public void initChannel(SocketChannel ch) throws Exception {
     public void test() throws Exception {
 
         io_uring_test();
-
+        Thread.sleep(1000);
```

Comment:
I added this one here to ensure that everything is teared down before starting again.